include/nuttx/net/netdev_lowerhalf.h

```diff
@@ -234,6 +258,21 @@ void netdev_lower_rxready(FAR struct netdev_lowerhalf_s *dev);
 
 void netdev_lower_txdone(FAR struct netdev_lowerhalf_s *dev);
 
+/****************************************************************************
+ * Name: netdev_lower_quota_load
+ *
+ * Description:
+ *   Fetch the quota, use this interface when device is running.
+ *
+ * Input Parameters:
+ *   dev  - The lower half device driver structure
+ *   type - Whether get quota for TX or RX
+ *
+ ****************************************************************************/
+
+int netdev_lower_quota_load(FAR struct netdev_lowerhalf_s *dev,
```

Comment:
What is the reason for exposing this interface? Is it going to be used by third party code in kernel?

Reply:
Yes, it's used here: https://github.com/apache/nuttx/pull/9297/files#diff-fa7cc8834e3748b89579efd930a809981c7267ef5aa0f573916ebe57ed1a2f71R319